app/src/main/java/com/app/communicationproject/activities/activities/MainActivity.java

```diff
@@ -0,0 +1,62 @@
+package com.app.communicationproject.activities.activities;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+
+import com.app.communicationproject.R;
+
+public class MainActivity extends BaseActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+
+    }
+
+    @Override
+    protected void onStart() {
+        super.onStart();
+    }
+
+    @Override
+    protected void onPause() {
+        super.onPause();
+    }
+
+    @Override
+    protected void onResume() {
+
+
+
+            super.onResume();
+
+
+    }
+
+    @Override
+    protected void onStop() {
+        super.onStop();
+    }
+
+    @Override
+    protected void onRestart() {
+        super.onRestart();
+    }
+
+    @Override
+    protected void onDestroy() {
+        super.onDestroy();
+    }
+
+
+    public void sendBroadcast(View v) {
+        Intent intent = new Intent();
+        intent.setAction("com.te.custombroadcast");
```

Comment:
Create Constants file & put it into that file. Example

public static final String ACTION_CUSTOM_BROADCAST = "com.te.custombroadcast";